rdt/transformers/pii/anonymizer.py

```diff
@@ -119,49 +102,17 @@ def _function(self):
         """Return a callable ``faker`` function."""
         return getattr(self.faker, self.function_name)(**self.function_kwargs)
 
-    def get_output_sdtypes(self):
-        """Return the output sdtypes supported by the transformer.
-
-        Returns:
-            dict:
-                Mapping from the transformed column names to supported sdtypes.
-        """
-        output_sdtypes = self.OUTPUT_SDTYPES
-        if self.null_transformer and self.null_transformer.models_missing_values():
-            output_sdtypes['is_null'] = 'float'
-
-        return self._add_prefix(output_sdtypes)
-
     def _fit(self, data):
         """Fit the transformer to the data.
 
         Args:
             data (pandas.Series):
                 Data to fit to.
         """
-        self.null_transformer = NullTransformer(
-            self.missing_value_replacement,
-            self.model_missing_values
-        )
-        self.null_transformer.fit(data)
         self.data_length = len(data)
 
-    def _transform(self, data):
-        """Return ``null`` column if ``models_missing_values``.
-
-        Args:
-            data (pandas.Series):
-                Data to transform.
-
-        Returns:
-            (numpy.ndarray or None):
-                If ``self.model_missing_values`` is ``True`` then will return a ``numpy.ndarray``
-                indicating which values should be ``nan``, else will return ``None``. In both
-                scenarios the original column is being dropped.
-        """
-        if self.null_transformer and self.null_transformer.models_missing_values():
-            return self.null_transformer.transform(data)[:, 1].astype(float)
-
+    def _transform(self, _data):
```

Comment:
`_data` is set in because of `W0613` : [How to fix](https://pycodequ.al/docs/pylint-messages/w0613-unused-argument.html)